src/EGPlanner/searchEnergy.cpp

```diff
@@ -356,6 +356,7 @@ SearchEnergy::potentialQualityEnergy(bool verbose) const
 	}
 
 	double gq = -1 ,gv = -1;
+    Q_UNUSED(gv)
```

Comment:
I would rather clean this up by removing gv as well as the debug statement that uses it altogether...